### PR TITLE
Import assets components

### PR DESCRIPTION
**Pull request by hank-1023**
- Import more colors from Zeplin (I exported colors from Zeplin and some of the naming might be changed. I will change it back if it breaks your code🤕🙇‍♂️).
- Changed Work-Sans Font file since iOS can't determine exact font sizing from current file.
- Added extension to "Font" for use in SwiftUI.
- Added "Feed" component.
- Added "TagSelected" component.

New updates this week:
- Added "Overlay", "Search Result" component.
- Added "Overlay Demo" for future reference.
- Updates build settings for Xcode 12.

### CODE REVIEW

File: FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//  FightPandemics
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
+//
+
+import SwiftUI
+
+struct SelectionSheetButton: View {
+    @State var selected: Bool = false
```
Comment (lucianoschillagi) [inline FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift:12]: A good practice is put "private" before "var"

File: FightPandemics/Views/Components/Cards/ExpandableText.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
+//
+
+import SwiftUI
+
+struct ExpandableText: View {
+    @State var text: String
+    @State var lineLimit: Int
```
Comment (lucianoschillagi) [inline FightPandemics/Views/Components/Cards/ExpandableText.swift:13]: A good practice is put "private" before "var"

File: FightPandemics/Views/Components/Cards/Feed.swift [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+struct Feed_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            Feed()
+                .previewLayout(.fixed(width: 395, height: 362))
+                .environment(\.colorScheme, .light)
+        }
+        
+    }
+}
```
Comment (lucianoschillagi) [inline FightPandemics/Views/Components/Cards/Feed.swift:117]: Good!
Maybe you can extract this UI pieces on differents structs-files (modularization)
Example: MetaInfoView, SocialButtonsView, etc

File: FightPandemics/Views/Components/Overlays/OverlayDemo.swift [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
+//
+
+import SwiftUI
+
+struct OverlayDemo: View {
+    @State var showOverlay = false
+    @State var dragTranslate = CGSize.zero
```
Comment (lucianoschillagi) [inline FightPandemics/Views/Components/Overlays/OverlayDemo.swift:13]: ...private

**Review by lucianoschillagi — COMMENTED** [review]
Please see:

- **private** var  (see the comments)
- modularization (see the comments)

Great work Hank!

File: FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift [hunk 5]
```diff
@@ -0,0 +1,32 @@
+//
+//  SelectionSheetButton.swift
+//  FightPandemics
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
```
Comment (stavares843) [inline FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift:6]: should it say FightPandemics where it says Scott?

File: FightPandemics/Views/Components/Overlays/OverlayDemo.swift (hunk 4 above)
Comment (Power186) [reply FightPandemics/Views/Components/Overlays/OverlayDemo.swift:13]: Yes, these need to be changed to private, great spot Luko

**Review by Power186 — CHANGES_REQUESTED** [review]
private var for @State please.  Other than great job.

File: FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift (hunk 5 above)
Comment (Power186) [reply FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift:6]: It's a Xcode generated comment from the original committed file I did when starting this.. we can delete all these on the files if we want, but it's just a comment generated from file creation.. do what you want 👍 as it makes no difference

**Review by Power186 — APPROVED** [review]
Nice changes, thanks for updating to private var..

**Review by Power186 — CHANGES_REQUESTED** [review]
Please delete viewModels as there are no models for views.  Please replace with extensions on View.

File: FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift (hunk 5 above)
Comment (lucianoschillagi) [reply FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift:6]: I understand. Ok, so we can to replace the files to "FightPandemics" as Copyright. Thanks Scott!

**Review by Power186 — CHANGES_REQUESTED** [review]
A view model for a toggle is not a pattern or practice.  Please add overlay as an extension, or possibly as a closure, to have that functionality shared across views, or put into a computed property.  Is this toggle style found in UX design?  Meaning is this consistent with the web app?  

Going from SearchResultViewModel to a model?  This will lead to complications later.  Can't have a view model without a model first.  Please fix.

**Pull request closed without merging**